packages/admin-panel/src/api/mutations/useEditSurveyResponse.js

```diff
@@ -19,6 +19,9 @@ export const useEditSurveyResponse = (surveyResponseId, updatedSurveyResponse) =
       onSuccess: async () => {
         // invalidate the survey response data
         await queryClient.invalidateQueries(['surveyResubmitData', surveyResponseId]);
+        if (onSuccess) {
+          onSuccess();
+        }
         return 'completed';
```

Comment:
```suggestion

```
I know it's not your change but do we need this return string? Seems a bit random